packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -846,6 +858,7 @@ class TooltipState extends State<Tooltip> with SingleTickerProviderStateMixin {
       verticalOffset:
           widget.verticalOffset ?? tooltipTheme.verticalOffset ?? _defaultVerticalOffset,
       preferBelow: widget.preferBelow ?? tooltipTheme.preferBelow ?? _defaultPreferBelow,
+      ignorePointer: widget.ignorePointer ?? widget.message != null,
```

Comment:
I was confused by this because the docs mention the default depending on `richMessage`, but it seems that richMessage must not be null when message is null. So all good.

Reply:
Cheers. I've changed the docs to have your "invisible" suggestion, and added "...instead of a [message]." so it's a bit easier to tie in with how the code is written. 

  /// Whether this tooltip should be invisible to hit testing.
  ///
  /// If no value is passed, pointer events are ignored unless the tooltip has a
  /// [richMessage] instead of a [message].
  ///
  /// See also:
  ///
  /// * [IgnorePointer], for more information about how pointer events are
  /// handled or ignored.

Reply:
That makes it more clear to me, good idea!